src/components/Footer/Footer.js

```diff
@@ -3,14 +3,13 @@ import * as React from 'react';
 import {StyleSheet, Text, TouchableOpacity, View} from 'react-native';
 
 import Toast from 'react-native-simple-toast';
-import MemoIc_arrow_down_vote_off from '../../assets/arrow/Ic_downvote_off';
-import MemoIc_arrow_down_vote_on from '../../assets/arrow/Ic_downvote_on';
-import MemoIc_arrow_upvote_off from '../../assets/arrow/Ic_upvote_off';
-import MemoIc_arrow_upvote_on from '../../assets/arrow/Ic_upvote_on';
+import LinearGradient from 'react-native-linear-gradient';
+import ArrowDownvoteOff from '../../assets/arrow/Ic_downvote_off';
+import ArrowDownvoteOn from '../../assets/arrow/Ic_downvote_on';
+import ArrowUpvoteOff from '../../assets/arrow/Ic_upvote_off';
+import ArrowUpvoteOn from '../../assets/arrow/Ic_upvote_on';
 import MemoIc_block_inactive from '../../assets/block/Ic_block_inactive';
-import MemoIc_comment from '../../assets/icons/Ic_comment';
 import MemoIc_share from '../../assets/icons/Ic_share';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been updated to reflect the new SVG components. The naming of these imports has also been simplified, which improves readability.
```diff
- import MemoIc_arrow_down_vote_off from '../../assets/arrow/Ic_downvote_off';
- import MemoIc_arrow_down_vote_on from '../../assets/arrow/Ic_downvote_on';
- import MemoIc_arrow_upvote_off from '../../assets/arrow/Ic_upvote_off';
- import MemoIc_arrow_upvote_on from '../../assets/arrow/Ic_upvote_on';
+ import ArrowDownvoteOff from '../../assets/arrow/Ic_downvote_off';
+ import ArrowDownvoteOn from '../../assets/arrow/Ic_downvote_on';
+ import ArrowUpvoteOff from '../../assets/arrow/Ic_upvote_off';
+ import ArrowUpvoteOn from '../../assets/arrow/Ic_upvote_on';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->